src/utils/cellSets/composeTree.js

```diff
@@ -6,13 +6,13 @@
  * all cell sets, including metadata.
  *
  */
-const composeTree = (hierarchy, properties, filterType = null) => {
-  const composeTreeRecursive = (data, type) => {
+const composeTree = (hierarchy, properties, filterTypes = null) => {
+  const composeTreeRecursive = (data, types) => {
     if (!data) {
       return;
     }
     return data.filter(
-      (root) => (!type || properties[root.key].type === type),
+      (root) => (!types || types.includes(properties[root.key].type)),
```

Comment:
will this be false or '[]'?

Reply:
I didn't understnad, which part are you asking about?